cluster-provision/k8s/1.17/provision.sh

```diff
@@ -34,29 +73,25 @@ repo_gpgcheck=1
 gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
        https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
 EOF
-yum install -y \
-  docker-1.13.1 \
-  python-docker-py-1.10.6 \
-  python3-pip
-
-pip3 install --default-timeout=100 docker-pycreds
-
-# Log to json files instead of journald
-sed -i 's/--log-driver=journald //g' /etc/sysconfig/docker
-echo '{ "insecure-registries" : ["registry:5000"] }' > /etc/docker/daemon.json
 
-# Enable the permanent logging
-# Required by the fluentd journald plugin
-# The default settings in recent distribution for systemd is set to auto,
-# when on auto journal is permament when /var/log/journal exists
-mkdir -p /var/log/journal
-
-# Omit pgp checks until https://github.com/kubernetes/kubeadm/issues/643 is resolved.
-yum install --nogpgcheck -y \
+# Install Kubernetes packages.
+dnf install --nogpgcheck --disableexcludes=kubernetes -y \
     kubeadm-${version} \
     kubelet-${version} \
     kubectl-${version} \
-    kubernetes-cni-0.6.0
+    kubernetes-cni
+
+# k8s repo is no longer needed and it introduce some
+# gpg key non interactive dnf install -y related to
+# gpg keys (using rpm --import does not fix it)
+rm /etc/yum.repos.d/kubernetes.repo
```

Comment:
Set [this](https://github.com/kubevirt/kubevirtci/pull/292/files#diff-4df819f2ca7dae6d02fae2a568d619fdR72) to '0' to prevent those issues

Reply:
ack let's try it out.